### PR TITLE
Add delta traffic stats

### PR DESCRIPTION
**Pull request by t123yh**
This is to solve #192.

Tested on my WR740n v3 with Gargoyle 1.7.1 installed.

Signed-off-by: t123yh <tianyh2000@163.com>

### CODE REVIEW

File: src/centralserver.c [hunk 1]
```diff
@@ -65,8 +65,9 @@
 */
 t_authcode
 auth_server_request(t_authresponse * authresponse, const char *request_type, const char *ip, const char *mac,
-                    const char *token, unsigned long long int incoming, unsigned long long int outgoing)
+                    const char *token, unsigned long long int incoming, unsigned long long int outgoing, unsigned long long int incoming_delta, unsigned long long int outgoing_delta)
 {
+		s_config *config = config_get_config();
```
Comment (acv) [inline src/centralserver.c]: Ideally that should be indented exactly 4 spaces.

**Comment by acv** [comment]
Seems fine to me. If you can remove all tab characters and use 4 spaces for all indent. Not a huge deal if you can't.

**Comment by t123yh** [comment]
Oh, sorry. I'll do it soon.

**Comment by t123yh** [comment]
@acv Is that alright now?

**Comment by acv** [comment]
Yep, looks good.

**Comment by t123yh** [comment]
@florida63 This is already added to the 1.3.0 milestone in #192.

**Comment by t123yh** [comment]
@florida63 Oh,  I see. You removed it. Sorry.

**Pull request merged**